src/skmultiflow/transform/missing_values_cleaner.py

```diff
@@ -1,197 +0,0 @@
-import numpy as np
-from scipy import stats
-from skmultiflow.transform.base_transform import StreamTransform
-from skmultiflow.utils.data_structures import FastBuffer
-from skmultiflow.utils.utils import get_dimensions
-
-
-class MissingValuesCleaner(StreamTransform):
```

Comment:
Instead of removing, use a deprecation warning to let the users know that this will be removed and the alternative (sklearn.impute)